src/filter.rs

```diff
@@ -234,6 +240,60 @@ pub fn emboss(mut src: &mut Image) -> RasterResult<()>{
     convolve(src, matrix, 1)
 }
 
+/// Apply sobel.
+///
+/// # Examples
+/// ```
+/// use raster::{filter, Orientation};
+///
+/// // Create image from file
+/// let mut image = raster::open("tests/in/sample.jpg").unwrap();
+/// filter::sobel(&mut image, Orientation::Horizontal).unwrap();
+/// raster::save(&image, "tests/out/test_filter_sobel.jpg").unwrap();
```

Comment:
To be consistent with the paths used in the tests, since `Orientation::Horizontal` was used, the correct output path is `"tests/out/test_filter_sobel_x.jpg"`